vp_functions.py

```diff
@@ -457,7 +450,20 @@ def altitude_parameter_averaging_cvp_static(radar, field, cvp_index, avg_range_d
     # get altitudes for the column values
     column_altitudes = altitudes[np.ix_(range(0,radar.nsweeps),az_indxs,r_indxs)] #13.12.2021 test
     #column_altitudes = altitudes[np.ix_(range(0,100),az_indxs,r_indxs)]
-
+    #get the voxel size for the altitude levels
+    
+    #### Neely
+    base = 5 # precision
+    
+    if h_step is None: h_step = int(1000 * round(((az_size)/1000))) 
```

Comment:
These values should never be None unless this function is called from outside the program without the correct arguments. If no max/min/step values are given on command line, the default values will be used, and these are the values that are passed to this function. As such, this code block may be unnecessary.

These lines should be preserved however is the function is likely to be called on its own as part of a library where something on the lines of `import vp_functions` is used. If that is the case, maybe have the `base` variable as something that can be changed without altering the code, for example an input argument to the function.